test/controllers/passwordless/sessions_controller_test.rb

```diff
@@ -154,7 +192,7 @@ class << User
       get "/users/sign_in/#{passwordless_session.token}"
       follow_redirect!
 
-      assert_equal "/", path
+      assert_equal Passwordless.success_redirect_path, path
```

Comment:
No need to change these if they already succeed.
```suggestion
      assert_equal "/", path
```